comps/dataprep/utils.py

```diff
@@ -457,3 +459,104 @@ def parse_html(input):
             print("The given link/str {} cannot be parsed.".format(link))
 
     return chucks
+
+
+def get_tables_result(pdf_path, table_strategy):
+    """Extract tables information from pdf file."""
+    if table_strategy == "fast":
+        return None
+
+    from unstructured.documents.elements import FigureCaption
+    from unstructured.partition.pdf import partition_pdf
+
+    tables_result = []
+    raw_pdf_elements = partition_pdf(
```

Comment:
Looks like it only support pdf, if this is the case, better add a note to explain this.